low-level/framework/utils/mon_utils.py

```diff
@@ -49,3 +50,19 @@ def normalize_kv(item, input_v, replace_v):
             return replace_v
         else:
             return item
+
+    @staticmethod
+    def sort_by_specific_kv(data, key_path, log):
+        """Accept list of dictionary and sort by key value."""
+        sorted_data = []
+        try:
+            if key_path and data:
```

Comment:
whats this keypath & how it looks exactly?

Reply:
Done